tools/getkeys/package-lock.json

```diff
@@ -4,6 +4,129 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@babel/code-frame": {
```

Comment:
Lot of new stuff here. Anyone know why?

Reply:
looks like f250b2914d3672d4a1492f72c6f0c3e3d42cd15a forgot to update the package lock file after adding linting, so this change seems like a good one to merge.